docs/user/rstudio/ide/guide/code/code-navigation.qmd

```diff
@@ -24,13 +22,13 @@ Double clicking the line will open the file in the RStudio source editor.
 
 ## Go to File/Function
 
-If you know the name of the source file or function that you want to edit next you can quickly navigate to it using the Go to File/Function search box on the main RStudio toolbar:
+If the name of the source file or function is known, the "Go to File/Function" search box on the main RStudio toolbar allows for quick navigation to it:
 
 ![GoToFile.png](images/go-to-function.png){fig-alt="A screenshot of the Go to File/Function menu item on the Source pane of RStudio."}
```

Comment:
Remove the `GoToFile.png` caption from this pic.

Reply:
removed